DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/CourseSetup/CourseContentViewModelTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.TrackingSystem.CourseSetup
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseContent;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    internal class CourseContentViewModelTests

```

Comment:
I thought the test classes are usually set up as public

Reply:
Happy to change it to public, but it looks like there are roughly 12 other test classes in Web.Tests that are set up as internal.